content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -0,0 +1,133 @@
+interested in our mission and scope.
+
+Many of you close to the ecosystem might be scratching your head - isn’t
+this what CNCF does?
+
+Yes and no. The CNCF has 40+ other projects that need to be marketed to
+a countless number of different types of community members in distinct
+ways and they aren’t responsible for the day to day operations of their
+projects. They absolutely do partner with us to highlight what we need
+and when we need it, and they do a fantastic job of it (one example is
+the [*@kubernetesio twitter account*](https://twitter.com/kubernetesio?lang=en) and its 200,000
+followers). 
+
+Where this working group differs is in its scope: we are entirely
+focused on elevating the hard work being done throughout the Kubernetes
+community by its contributors.
+
+## What to expect from us
+
+You can expect to see us in our web of [communication channels](https://github.com/kubernetes/community/tree/master/communication) supporting you by:
+
+  - Finding ways of adding our human touch to potentially overwhelming
+    quantities of info by storytelling and other methods - we want to
+    highlight you and provide useful information!
+  - Providing ways to clarify expectations on what’s important, urgent,
+    or otherwise
+  - Keeping you in the know of the comings and goings of contributor
+    community events, activities, mentoring initiatives, KEPs, and more.
+  - Creating a presence on Twitter specifically for contributors via
+    @k8scontributors that is all about being a contributor in all its
+    forms.
+
+What does this look like in the wild? [Our first post](https://kubernetes.io/blog/2020/03/19/join-sig-scalability/) in a series about our 40 community groups landed recently. Did you see it?
+More articles like this and additional themes of stories to flow through
+[our storytellers](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose).
+
+We will deliver this with an [ethos](https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md#ethosvision) behind us aligned to the Kubernetes project as whole, and we’re
+committed to using the same tools as all the other SIGs to do so. Check out our [project board](https://github.com/orgs/kubernetes/projects/39) to view our roadmap of upcoming work.
+
+## Join us and be part of the story
+
+This initiative is early on and we still have important roles to fill to
+make it successful. 
+
+If you are interested in open sourcing marketing functions – it’s a neat
+ride – join us! Specific immediate roles include storytelling through
+blogs and designers and anything in progress on our project board. 
+
+Also, if you’re reading this, you’re exactly the type of person we are
+here to support. Want to give us feedback or suggestions? Missed
+something in the past that should have been highlighted? Contributor
+that wants something you are working on to be highlighted? We would love
+to hear about how to improve. 
```

Comment:
could we nix the last line? if we put the ending statement with the missed something in the past bit and end it there it will flow a bit better.